specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4366,13 +4366,24 @@
       "description": "settings for security policy patterns to match",
       "type": "object",
       "properties": {
+        "IsProfileLevel": {
```

Comment:
We may want to consider naming this property as "Default" or something similar. It depends a bit on expected behavior for the WAF policy (needs discussion with PMs).